pages/availability/event/[type].tsx

```diff
@@ -29,6 +35,32 @@ export default function EventType(props) {
     const isHiddenRef = useRef<HTMLInputElement>();
     const eventNameRef = useRef<HTMLInputElement>();
 
+    const [locationOptions, setLocationOptions] = useState(DEFAULT_LOCATION_OPTIONS)
```

Comment:
I don't think this will ever dynamically update, so perhaps instead of useState this should be part of the getServerProps?